.travis.yml

```diff
@@ -28,15 +28,15 @@ before_script:
 
 script:
    - |
-       travis-cargo --only nightly test -- --features nightly --no-default-features &&
+       travis-cargo --only nightly test -- --no-default-features &&
```

Comment:
We probably don't need this at all now, in theory.  The same build process should work for `stable`, `beta`, and `nightly`.